sdk/internal/recording/recording.go

```diff
@@ -84,7 +85,12 @@ const (
 	Secret_Base64String VariableType = "secret_base64String"
 )
 
+// defaultSanitizersSet tracks whether default sanitizers have been added (this would be neater with Go 1.19's atomic.Bool)
```

Comment:
Lets folow up and move to 1.19. Afterall 1.22.2 and 1.21.9 are the only in support versions on golang.org

Reply:
Will there be a ripple effect, requiring us to update everything? Might be best to do that separately.